packages/components/src/Glimmer/Glimmer.stories.mdx

```diff
@@ -33,13 +34,52 @@ import { Glimmer } from "@jobber/components/Glimmer";
       timing: "base",
     }}
   >
-    {args => <Glimmer {...args} />}
+    {args => (
+      // Avoid storybook wrapping flex messing with square and circle example works
+      <div>
+        <Glimmer {...args} />
+      </div>
+    )}
   </Story>
 </Canvas>
 
 <ArgsTable of={Glimmer} story="Glimmer" />
 
-## Example
+## Colors
+
+Glimmers adapts to the background you put it in. If you have a grey background,
+the `Glimmer` will automatically darken itself so it's still noticeable.
+
+<Canvas style={{ background: "var(--color-surface--background)" }}>
+  <Glimmer />
+</Canvas>
+
+If you have a darker or coloured background, you can toggle the `reverseTheme`
```

Comment:
Why `reverseTheme` and not `darkTheme`? Mostly curious, but I think Dark Mode/Theme is more standard...

Reply:
no idea. was making it consistent with the mobile text prop 
![image](https://github.com/GetJobber/atlantis/assets/15986172/2a8e8eb4-6a2f-4998-b92a-d279d74ef163)

Reply:
Although I understand why `darkTheme` sounds more familiar, `--color-surface--reverse` connects to `reverseTheme`, pointing to that colour rather than, just black.

Reply:
Ok, makes sense. 👍🏻